### ISSUE TITLE
Landing Pages Keep Getting Flagged by Google As Spam

### ISSUE THREAD

**Issue by teddielong** [gophish/gophish#802]
Hi everyone.

I'm helping a few charities to conduct simulated phishing (I'm external to their network) and overall, GoPhish is amazing but what I am struggling with is to maintain the same domain from which to launch the exercises.

For example, I set up GoPhish Domain1.com using an email server to send the emails, with a landing page for domain1.com.  However, after a few campaigns, Google is automatically showing Domain1.com to be a suspected phishing site, so I'm having to set up domain2.com and start all over again.

Has anyone else had this problem?  I appreciate this is a Google thing, but just curious as to whether there is a workaround I'm missing to prevent me needing to whip up a new domain every other campaign?

Cheers

**Comment by stracire** [comment]
Did you use a Google product phishing site? I discovered that Chrome does scan URL's and pages for Google product related pages. The workaround for me is to not use Google related pages.

**Comment by teddielong** [comment]
This happens when I mimmick a Amazon log in page (although it works fine for a few campaigns and then starts to display the warning screen) - maybe this Amazon one is the issue?  Also had the same problem with an Apple landing page, but not LinkedIn?

The URL being used has nothing to do with Amazon, but I did import the site via the HTML tool - but this is exactly what I did with LinkedIn and that seems to work ok?

How do, for example KnowBe4 get around this issue? They seem to have a fixed domain for this rather than dotting around with new domains all the time.  I

**Comment by jordan-wright** [comment]
There's quite a few things that can contribute to this:

* Reputation with service provider - Companies that do this absolutely have tons of domains they can churn through. They communicate these domains to industry partners to try and make it clear that they are a phishing simulation provider and not an actual threat.

* Lack of branding - Most of the time, the phishing page will be unbranded to avoid clear trademark violations. This is a big reason why I am very hesitant to accept phishing templates for built-in use with Gophish

* Transparent landing page - Gophish just gives a 404 page on the index page, while many phishing service providers will give an informational page about who is running the service. I am thinking about updating this for Gophish, fwiw. But, this would only tell network admins that it's a Gophish instance, not _who's_ running the instance.

In a nutshell, it's really hard to give the same guarantees for Gophish that you get with a centralized saas model. There's all sorts of things you _could_ do to try and hide the site but that gets sketchy really fast.

My best recommendation is to be as transparent as possible with your Gophish instance, as well as trying to set things up the best you way can (spf, dmarc, dkim, https, etc.). You could also consider restricting access to your Gophish instance to only the IP addresses you expect users to come from (this'll be tricky if people are mobile).

That's about the best we can do, I'm afraid. Hope this helps!
